### ISSUE TITLE
Update to text node within aria-live not reading out on IE11 and Edge. 

### ISSUE THREAD

**Issue by thomasoreilly** [FreedomScientific/VFO-standards-support#408]
### Summary

When implementing a live region and updating text on a text node, JAWS doesn't read out the update. 

### Expected result

Read out occurs. 

### Actual result

No read out occurs 

### Example

I've created a Stackblitz to demo the issue. 
https://stackblitz.com/edit/js-udtrgw?file=index.html

### Additional Information

This has a big impact on Angular applications as this is the method of updating text. 

#### JAWS version and build number

Version: 2020.2006.12 ILM
 
#### Operating System and version

Windows 10
Version: 10.0.17763 Build 17763

#### Browser and version:

IE11 
Version: 11.1339.17763.0

Edge 
Version: 44.17763.831.0

**Comment by LauraOH** [comment]
Element with aria-live has to be present in the page before the contents are updated.

**Comment by thomasoreilly** [comment]
Hey @LauraOH. 
The contents are present before the content is updated.
The issue is the text node is updated for the p tag. 

Playing around with it a bit more here and I noticed that using a span with aria-live directly on that span and updating the span's text node works. This is a little inconvenient when looking to read out text changes of grand children relative to the aria-live element.

**Comment by JAWS-test** [comment]
With an older version of JAWS (JAWS 2020.2003) and IE 11, the live region is output correctly

**Comment by thomasoreilly** [comment]
Hi @JAWS-test. 

I have found a workaround to this issue by using span and aria-live. 
I have updated the example linked in this post to reflect the fix. 

I am happy to close this issue unless further investigation is needed on your end to figure out why aria-live only works with spans.

**Comment by JAWS-test** [comment]
The actual JAWS problem is not solved by the work around. Therefore, I would ask you to leave the issue open and republish your original example that did not work

**Comment by thomasoreilly** [comment]
Okay I will leave the closing to you. The example with the broken code is still there, see brokenExample refs.

**Comment by JAWS-test** [comment]
Looks like a duplicate of https://github.com/FreedomScientific/VFO-standards-support/issues/38.

**Comment by thomasoreilly** [comment]
Reviewing the Codepen I believe that is the same issue.
